lib/src/signed_video_h26x_sign.c

```diff
@@ -641,11 +640,10 @@ signed_video_get_nalu_to_prepend(signed_video_t *self,
     } else {
       nalu_to_prepend->prepend_instruction = SIGNED_VIDEO_PREPEND_NOTHING;
     }
-    return svi_rc_to_signed_video_rc(
-        get_latest_sei(self, nalu_to_prepend->nalu_data, &nalu_to_prepend->nalu_data_size));
-  } else {
+    status = get_latest_sei(self, nalu_to_prepend->nalu_data, &nalu_to_prepend->nalu_data_size);
```

Comment:
This should only be called if `*sei_size > 0`

Reply:
Would it work if I call it  while I am checking  if the |sei_size != 0|?